bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "angular-ts-utils",
-  "version": "0.2.4",
+  "version": "0.3",
```

Comment:
can you put a complete semver version? 0.3.0. Just to make sure that no tools will have issues with it!